client/src/components/Invite/Invite.scss

```
@@ -0,0 +1,21 @@
+@import 'styles/variables';
+@import 'styles/helpers';
+
+.Invite {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+  &__LinkBlock {
+    display: flex;
+    margin-top: $spacing-small;
+    align-items: center;
+    [type='text'] {
+      flex-basis: 75%;
```

Seems a bit arbitrary...
You can instead use `flex-grow: 1` if you want the "input" to fill all the available space
```suggestion
      flex-grow: 1;
```